docs/src/tutorials/multi.qmd

```diff
@@ -67,26 +100,67 @@ fit!(la, data)
 optimize_prior!(la; verbose=true, n_steps=100)
 ```
 
+with either the probit approximation:
+
 ```{julia}
 #| output: true
 
 _labels = sort(unique(y))
 plt_list = []
 for target in _labels
-    plt = plot(la, X, y; target=target, clim=(0,1))
+    plt = plot(la, X_test, y_test; target=target, clim=(0,1))
     push!(plt_list, plt)
 end
 plot(plt_list...)
 ```
 
+ or the plugin approximation:
+
 ```{julia}
 #| output: true
 
 _labels = sort(unique(y))
 plt_list = []
 for target in _labels
-    plt = plot(la, X, y; target=target, clim=(0,1), link_approx=:plugin)
+    plt = plot(la, X_test, y_test; target=target, clim=(0,1), link_approx=:plugin)
     push!(plt_list, plt)
 end
 plot(plt_list...)
-```
\ No newline at end of file
+```
+
+## Calibration Plots
+
+In the case of multiclass classification tasks, we cannot plot directly the calibration plots since they can only be used in the binary classification case. However, we can use them to plot the calibration of the predictions for 1 class against all the others. To do so, we first have to collect the predicted categorical distributions
```

Comment:
"we cannot plot directly the calibration plots" -> "we cannot plot the calibration plots directly"